src/main/resources/assets/js/app/highchart.es6

```diff
@@ -43,85 +43,93 @@ const createSetOptions = {
+            '{name}, serie {ix} av {numSeries}. Stolpediagram med {numPoints} {#plural(numPoints, bars, bar)}.',
+          default: '{name}, serie {ix} av {numSeries} med {numPoints} data {#plural(numPoints, points, point)}.',
+          defaultCombination:
+            '{name}, serie {ix} av {numSeries} med {numPoints} data {#plural(numPoints, points, point)}.',
+          line: '{name}, linje {ix} av {numSeries} med {numPoints} data {#plural(numPoints, points, point)}.',
+          lineCombination:
+            '{name}, serie {ix} av {numSeries}. Linje med {numPoints} data {#plural(numPoints, points, point)}.',
+          map: '{name}, kart {ix} av {numSeries} med {numPoints} {#plural(numPoints, areas, area)}.',
+          mapbubble: '{name}, Boblediagram {ix} av {numSeries} med {numPoints} {#plural(numPoints, bubbles, bubble)}.',
+          mapbubbleCombination:
+            '{name}, serie {ix} av {numSeries}. Boblediagram serie med {numPoints} {#plural(numPoints, bubbles, bubble)}.',
+          mapCombination: '{name}, serie {ix} av {numSeries}. Kart med {numPoints} {#plural(numPoints, areas, area)}.',
+          mapline: '{name}, linje {ix} av {numSeries} med {numPoints} data {#plural(numPoints, points, point)}.',
+          maplineCombination:
+            '{name}, serie {ix} av {numSeries}. Linje med {numPoints} data {#plural(numPoints, points, point)}.',
+          pie: '{name}, Kake {ix} av {numSeries} med {numPoints} {#plural(numPoints, slices, slice)}.',
+          pieCombination: '{name}, serie {ix} av {numSeries}. Kake med {numPoints} {#plural(numPoints, slices, slice)}.',
+          scatter: '{name}, spredningsplott {ix} av {numSeries} med {numPoints} {#plural(numPoints, points, point)}.',
+          scatterCombination:
+            '{name}, serie {ix} av {numSeries}, spredningsplott med {numPoints} {#plural(numPoints, points, point)}.',
+          spline: '{name}, linje {ix} av {numSeries} med {numPoints} data {#plural(numPoints, points, point)}.',
+          splineCombination:
+            '{name}, serie {ix} av {numSeries}. Linje med {numPoints} data {#plural(numPoints, points, point)}.',
+        },
       },
+      credits: 'Diagramkreditter: {creditsStr}.',
+      svgContainerLabel: 'Interaktiv graf',
+      defaultChartTitle: 'Graf',
+      contextButtonTitle: 'Last ned/skriv ut',
+      decimalPoint: ',',
+      downloadJPEG: 'Last ned som JPEG',
+      downloadPDF: 'Last ned som PDF',
+      downloadPNG: 'Last ned som PNG',
+      downloadSVG: 'Last ned som SVG',
+      downloadCSV: 'Last ned tala som CSV',
+      downloadXLS: 'Last ned tala som XLS',
+      drillUpText: 'Tilbake til {series.name}',
+      loading: 'Tegner graf...',
+      noData: 'Tall ikke tilgjengelig',
+      numericSymbols: [null, ' mill.', ' mrd.'],
+      printChart: 'Skriv ut graf',
+      resetZoom: 'Nullstill zoom',
+      resetZoomTitle: 'Nullstill zoom',
+      thousandsSep: ' ',
     },
-    svgContainerLabel: 'Interaktiv graf',
```

Comment:
Dont move this, only chartTypes and Series

Reply:
Har flyttet tilbake :) men jeg tror disse 3 må ligge under 'accessibility': 

```
credits: 'Diagramkreditter: {creditsStr}.',
svgContainerLabel: 'Interaktiv graf',
defaultChartTitle: 'Graf',
```

Reply:
https://api.highcharts.com/highcharts/lang.accessibility